homeassistant/helpers/entity.py

```diff
@@ -151,7 +170,13 @@ def update_ha_state(self, force_refresh=False):
                 "No entity id specified for entity {}".format(self.name))
 
         if force_refresh:
-            self.update()
+            if hasattr(self, 'async_update'):
```

Comment:
Is there a good reason for having a separate method for 'async_update'? Could we inspect the decorators using `_decorators` instead?